improver_tests/psychrometric_calculations/air_parcel_temperature_at_pressure_level/test_TemperatureSaturatedAirParcel.py

```diff
@@ -154,6 +154,23 @@ def test_basic(temperature, pressure, air_parcel):
     assert np.isclose(result.data, air_parcel.data, atol=1e-2).all()
 
 
+def test_masked(temperature, pressure, air_parcel):
+    """Check that masked inputs result in masked outputs."""
```

Comment:
The output of this test is not masked, nor is there a check to see if it is masked.